src/figuresClasses.ts

```diff
@@ -1,11 +1,87 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape: string;
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
```

Comment:
[CODE KNOWLEDGE] You don't need to manually assign properties for constructor parameters that have an access modifier. TypeScript does this automatically. Consider removing the manual assignments in the constructor body.